src/Shared/HandyControl_Shared/Media/Animation/GeometryAnimationUsingKeyFrames.cs

```diff
@@ -38,7 +38,15 @@ public string[] Strings
         
         private bool _areKeyTimesValid;
 
-        public GeometryAnimationUsingKeyFrames() => _areKeyTimesValid = true;
+        public GeometryAnimationUsingKeyFrames()
+        {
+            var currentCulture = System.Threading.Thread.CurrentThread.CurrentCulture.Parent.Name;
+            if (!currentCulture.Equals("en", StringComparison.OrdinalIgnoreCase))
+            {
+                System.Threading.Thread.CurrentThread.CurrentCulture = new System.Globalization.CultureInfo("en-US");
+            }
+            _areKeyTimesValid = true;
+        }
```

Comment:
So these codes means that as long as you don't use en, you default to en-us? I don't know if it's going to have any side effects while the #184 can be solved.

Reply:
ok, you put codes in GeometryAnimationUsingKeyFrames's ctor but what about GeometryAnimation?

Reply:
It works well as you can see
However, I don't know if GeometryAnimation needs to change or not
![1](https://user-images.githubusercontent.com/9213496/69820797-30c53600-1217-11ea-9b30-931712200747.gif)

Reply:
GeometryAnimationUsingKeyFrames was be used in demo, you can try GeometryAnimation too.

Reply:
I tested the following code and it worked
 `System.Threading.Thread.CurrentThread.CurrentCulture = CultureInfo.InvariantCulture;`
I think it's better than the previous code
I also added this code to GeometryAnimation But there was no example in the demo to test it

Reply:
Hi again
I tested with the following code and it worked smoothly without `System.Threading.Thread.CurrentThread.CurrentCulture = CultureInfo.InvariantCulture;
` in GeometryAnimation

```
 <Storyboard x:Key="StoryboardOnLoaded2" RepeatBehavior="Forever" AutoReverse="True">
            <hc:GeometryAnimation From="{StaticResource FaceBookGeometry}" To="{DynamicResource TwitterGeometry}" Storyboard.TargetProperty="Data" Storyboard.TargetName="PathDemo2">
                <hc:GeometryAnimation.EasingFunction>
                    <QuarticEase EasingMode="EaseInOut"/>
                </hc:GeometryAnimation.EasingFunction>
                
            </hc:GeometryAnimation>
        </Storyboard>

            <Path Name="PathDemo2" Width="100" Height="100" Data="{StaticResource TwitterGeometry}" Fill="#3b5998"/>

```
![1](https://user-images.githubusercontent.com/9213496/69825051-21e48080-1223-11ea-9be2-634799c4ed01.gif)